txauthor.go

```diff
@@ -35,12 +35,17 @@ func (tx *TxAuthor) SetSourceAccount(accountNumber int32) {
 	tx.sendFromAccount = uint32(accountNumber)
 }
 
-func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool) {
+func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool, wallet *Wallet) (verifyAddress bool) {
+	_, err := dcrutil.DecodeAddress(address, wallet.chainParams)
+	if err != nil {
+		return
+	}
 	tx.destinations = append(tx.destinations, TransactionDestination{
 		Address:    address,
 		AtomAmount: atomAmount,
 		SendMax:    sendMax,
 	})
+	return
```

Comment:
As explained above, returning an explicit value makes more readable sense. By the way, the 2 returns in this method would return false since the return variable `verifyAddress` was never modified - it's value remained false.
```suggestion
	return true
```

Reply:
The changes have been effected